source/features/prevent-pr-commit-link-loss.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import AlertIcon from 'octicon/alert.svg';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '../libs/features';
+import {prCommitRegex, preventPrCommitLinkBreak} from '../libs/utils';
+
+function handleButtonClick(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
+	const field = event.delegateTarget.form!.querySelector('textarea')!;
+	textFieldEdit.set(field, preventPrCommitLinkBreak(field.value));
+	field.parentElement!.remove();
+}
+
+function handleInput(event: delegate.Event<InputEvent, HTMLTextAreaElement>): void {
+	const field = event.delegateTarget;
+
+	if (prCommitRegex.test(field.value) && !select.exists('.rgh-fix-pr-commit-links')) {
```

Comment:
Shouldn't the `!select.exists('.rgh-fix-pr-commit-links')` check in that text box only? since you can have multiple textareas? for example when reviewing a PR?